.planning/handovers/HDR_COLOR_FIX_PLAN.md

```
@@ -0,0 +1,643 @@
+**File**: `engine/src/Utils/ColorUtils.ts`
+
+Add ACES Filmic approximation function:
+```ts
+function acesFilmic(x: number): number {
+    const a = 2.51, b = 0.03, c = 2.43, d = 0.59, e = 0.14;
+    return clamp((x * (a * x + b)) / (x * (c * x + d) + e), 0, 1);
+}
+```
+
+Apply in `getHdrStyleFromRgb` before CSS output:
+```ts
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)